doc/character_selector.rdoc

```diff
@@ -0,0 +1,64 @@
+== Character Selectors
+
+A _character_ _selector_ is a string argument accepted by certain Ruby methods.
+Each of these instance methods accepts one or more character selectors:
+
+- String#tr(selector, replacements): returns a new string.
+- String#tr!(selector, replacements): returns +self+.
+- String#tr_s(selector, replacements): returns a new string.
+- String#tr_s!(selector, replacements): returns +self+.
```

Comment:
Method definitions with multiple arguments don't render quite right compared to methods with single arguments. I think either we don't include method arguments here or manually link to the method definitions so that the links render consistently.

<img width="550" alt="Screen Shot 2022-03-08 at 2 28 45 PM" src="https://user-images.githubusercontent.com/15860699/157310869-dc2e55b4-eccb-4a0c-8685-22bad555b764.png">

Reply:
These links all work for me.  If the arguments' not rendering properly is a problem, I think it's an Rdoc problem.  If the links work, I vote to keep the arguments;  I think they clarify the semantics of the calls, especially the top four.

Reply:
Good point. I've opened ruby/rdoc#871 which fixes this problem. Using that rdoc to build the HTML, it looks much better.

<img width="589" alt="Screen Shot 2022-03-08 at 5 09 43 PM" src="https://user-images.githubusercontent.com/15860699/157334189-977726f5-1d90-499c-b9b9-fa252b8d6cdc.png">